include/curl/multi.h

```diff
@@ -187,6 +188,15 @@ CURL_EXTERN CURLMcode curl_multi_poll(CURLM *multi_handle,
                                       int timeout_ms,
                                       int *ret);
 
+/*
+ * Name:     curl_multi_unblock()
+ *
+ * Desc:     unblocks a "hanging" curl_multi_poll call.
```

Comment:
Might improve the description, as `unblocks a "hanging" curl_multi_poll call.` - sounds like a call that hung (eg deadlocked).

Reply:
The "unblock" word itself can sound a bit horrifying I chose this because it was already in the TODO sectiom, personally I'd prefer to call this "wakeup". The description then could be something like " wakes up a sleeping curl_multi_poll call". Ideas? Or if we leave it named unblock, what description should we use?

Reply:
Naming is hard. I don't have any problems with renaming it `*_wakeup` if you think that's better.

Is there any previous art somewhere for a similar API that we could copy the name from?

Reply:
I was mainly referring to `"hanging"` in the description. not the `_unblock` part. Description perhaps should make it clear that the api unblocks/wake up another thread that's blocked on/waiting for curl_multi_poll calls

Reply:
Now it is called `curl_multi_wakeup` and the short description says `wakes up a sleeping curl_multi_poll call`, I hope this is better.